recipes/lightgbm/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "lightgbm" %}
+{% set version = "2.0.4" %}
+{% set sha256 = "6472ded5b0df03504b9d53f4d0e8070a59c2ea015ae526137305c75eb2a77eec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - cmake_osx.patch  # [osx]
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt  # [not osx]
+  script: export LDFLAGS="${LDFLAGS} -L$PREFIX/lib"; python setup.py install --single-version-externally-managed --record record.txt  # [osx]
+  skip: True  # [win32]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cmake >=3.8
+    - libcxx  # [osx]
+    - openmp  # [osx]
```

Comment:
I'm not really sure about what might change from minor release to minor release of libcxx and openmp. Also, is there any danger with having `libcxx` as a runtime requirement in terms of conflicts with other packages?